content/docs/ref/comment.md

```diff
@@ -49,30 +49,6 @@ Any [generic option](/doc/ref) in addition to:
 
 ## Examples
 
-### Creating commit or issue comments
```

Comment:
removing because doesn't seem to explain anything more complex beyond the existing bullet above